frontend/src/components/MapView/utils.ts

```diff
@@ -153,6 +153,13 @@ export const filterActiveLayers = (
   selectedLayer: LayerType,
   categoryLayer: LayerType,
 ): boolean | undefined => {
+  // Special case for tropical storms `adamts_buffers`
+  if (
+    selectedLayer.id.includes('adamts') &&
+    selectedLayer.id.includes('adamts')
+  ) {
```

Comment:
You mean categoryLayer for the second one?
Can you explain why this is a special case? Ideally, we want to avoid named exceptions like this.

Reply:
Yes because the Tropical Storms is not one layer with group, but it is splitted into 3 different layers the default first one has a group and the other 2 have not, and when those are going to be compared they have different ids so they are going to be compared with the function `filterActiveGroupedLayers`, but because the 2 tropical storms layers will not have a group will be listed as different layers so the summary of the functions will going to give us 3 selected layers instead of 1

Reply:
@HarryMytilinaios I also meant that right now this is testing twice for the same thing `selectedLayer.id.includes('adamts')`

Reply:
You are right I will remove it and investigate a little bit more.

Reply:
Done